src/PKSim/Program.cs

```diff
@@ -23,7 +23,7 @@ private static void Main(string[] args)
          Application.EnableVisualStyles();
          Application.SetCompatibleTextRenderingDefault(false);
 
-         WindowsFormsSettings.SetDPIAware();
+      //   WindowsFormsSettings.SetDPIAware();
```

Comment:
What is this change about?

Reply:
SetDPIAware redners the app unsuable in High Res screens. I was wondering why PKsim looked like shit and MoBi not on my 4k monitor (laptop). That's the reason.  I believe this flag was used for the wrong reason and I can't remember why we added it (not in MoBi for instance)

Reply:
Issue created: #1148
I think this could easily happen when users create observers with commonly used names e.g. _TotalAmount_, _AverageConcentration_ etc.

Reply:
But they would have to want to load the SAME observer in the same buildign block. Why would they want to do that

Reply:
E.g. user has defined 2 observers in 2 different Mobi projects; same name but different functionality E.g.
* project1: _TotalAmount_ (which sums up everything in _Kidney_)
* project2: _TotalAmount_ (which sums up everything in _Liver_)
Now user wants to have both in PK-Sim

Reply:
Not the show stopper if it's not done, so let ask the users.

Reply:
IT;'s easy. It needs to be in different building blocks! And if they want to use it in the same simulation, well they should name stuff propertly